phabfive/maniphest.py

```diff
@@ -635,31 +640,37 @@ def _print_priority_transitions(self, priority_transactions, indent="  "):
                     elif new_order > old_order:  # Lowered (lower priority)
                         direction = "[↓]"
 
-            print(
-                f'{indent}- "{timestamp_str} {direction} {old_priority_name} → {new_priority_name}"'
+            transitions.append(
+                f"{timestamp_str} {direction} {old_priority_name} → {new_priority_name}"
             )
 
-    def _print_transitions(self, transactions, column_info, indent="      "):
+        return transitions
+
+    def _build_transitions(self, transactions, column_info):
```

Comment:
Rename to `_build_column_transitions()`?

Reply:
changed